pkg/extender/extender.go

```diff
@@ -107,8 +107,9 @@ func (e *Extender) Start() error {
 		return fmt.Errorf("Extender has already been started")
 	}
 	// TODO: Make the listen port configurable
-	e.server = &http.Server{Addr: ":8099"}
-	http.HandleFunc("/", e.serveHTTP)
+	mux := http.NewServeMux()
+	mux.HandleFunc("/", e.serveHTTP)
+	e.server = &http.Server{Addr: ":8099", Handler: mux}
```

Comment:
curious about the intent of this change and what does the new change do differently?

Reply:
The explanation was here https://github.com/libopenstorage/stork/pull/1838#issuecomment-2322069879 . However , dropped the idea of creating separate test suite with cache enabled and added the test in the same test suite which means no need to run the extender server again, thus avoiding the error 
```
panic: http: multiple registrations for /
```